components/test/brave_extension/state/shieldsPanelState_test.ts

```diff
@@ -196,6 +226,9 @@ describe('shieldsPanelState test', () => {
     it('updates the currentWindowId', () => {
       expect(shieldsPanelState.updateFocusedWindow(state, 2)).toEqual({
         currentWindowId: 2,
+        persistentData: {
```

Comment:
nit: this all seems quite repetitive! Perhaps a function `getDefaultState` would remove the need to copy and keep in sync amongst the tests, next time we need to update this :-)

Reply:
are you ok if we address this in a follow-up? I've been looking to re-work some shields tests for a while, can add this as well

Reply:
Absolutely, it was just a comment